### DISCUSSION TITLE
大廳字形整合問題

### DISCUSSION THREAD

**Discussion by qwer20108** [GpointChen/FFXIVChnTextPatch-GP#161] [General]
請問有沒有打算改字形檔, 以顯示大廳翻譯

我花了點時間研究一下這篇文章後
https://home.gamer.com.tw/artwork.php?sn=5399504

安裝了 XivAlexander 並且依照預設的 SourceHanSansSC.json 安裝思源黑體字形進去後大廳翻譯終於正確顯示了.

請問之後有沒有打算乾脆直接將以 XivAlexander 安裝過的字型檔由 000000.win32.index 抽出來放到 font 資料夾直接使用?

**Comment by GpointChen** [comment]
確實是有，不過用這個產生的字體檔普遍偏大，還在研究如何壓縮

**Reply by qwer20108** [reply]
我剛玩FF14 2.0 主線還沒解完 一直看怎麼漢化
我看 XivAlexander  會將字型檔打包後放在 %AppData%\XivAlexander\Cached 中當載入遊戲時一起載入記憶體不會 patch 到 000000.win32.index

我把裡面的 
TTMPL.mpl
TTMPD.mpd
打包成 SourceHanSansSC.zip 後以 [FFXIV_TexTools_UI](https://github.com/TexTools/FFXIV_TexTools_UI) 開啟可看到字體檔現在正在研究怎麼把TTMP內的檔案抽出來, 但是好像沒有類似工具可以抽取目前正在看 [LoadFromButton_Click](https://github.com/TexTools/FFXIV_TexTools_UI/blob/master/FFXIV_TexTools/Views/ModPack/Wizard/ModPackWizard.xaml.cs#L352) 這個 function 研究抽取檔案的方法.

**Reply by GpointChen** [reply]
如上文連結內所述，用ffxiv launcher配合penumbra就可以提取出來了。

**Reply by qwer20108** [reply]
感謝找到了. 
我還去看了 XivAlexander 程式碼想研究怎麼產生字體把字體抽出來, 沒寫過 C++ vcpkg 弄了很久.
